inst/tests/tests.Rraw

```diff
@@ -21141,3 +21141,13 @@ dt = data.table(id = 1:25)
 test(2314.1, any(grepl("<int>", tail(capture.output(print(dt, class = TRUE)), 2))), TRUE)
 # Test that class=TRUE with col.names="top" doesn't show classes at bottom
 test(2314.2, !any(grepl("<int>", tail(capture.output(print(dt, class = TRUE, col.names = "top")), 2))), TRUE)
+
+# Test that tzone behavior stops warning with ignore_tzone=TRUE
+tm1 = .POSIXct(1, "UTC")
```

Comment:
nit: the structure of the test is odd, why put `a`/`b` in a data.table, but not `tm1`?

I would put either all or no `between()` inputs in the same table.